superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.jsx

```diff
@@ -68,38 +58,41 @@ class ExploreCtasResultsButton extends React.PureComponent {
           all_columns: [],
           row_limit: 1000,
         };
-        this.props.actions.addInfoToast(
+        props.actions.addInfoToast(
           t('Creating a data source and creating a new tab'),
         );
 
         // open new window for data visualization
         exploreChart(formData);
       })
       .catch(() => {
-        this.props.actions.addDangerToast(
-          this.props.errorMessage || t('An error occurred'),
+        props.actions.addDangerToast(
+          props.errorMessage || t('An error occurred'),
         );
       });
   }
 
-  render() {
-    return (
-      <>
-        <Button
-          buttonSize="small"
-          onClick={this.onClick}
-          tooltip={t('Explore the result set in the data exploration view')}
-        >
-          <InfoTooltipWithTrigger
-            icon="line-chart"
-            placement="top"
-            label="explore"
-          />{' '}
-          {t('Explore')}
-        </Button>
-      </>
-    );
+  function onClick() {
+    visualize();
   }
+
+  return (
+    <>
```

Comment:
this extra fragment was left over from the original code, but you can remove it.